src/client/pythonEnvironments/info/index.ts

```diff
@@ -14,35 +14,33 @@ type ReleaseLevel = 'alpha' | 'beta' | 'candidate' | 'final' | 'unknown';
  */
 export type PythonVersionInfo = [number, number, number, ReleaseLevel];
 
+/**
+ * The supported Python environment types.
+ */
+export enum EnvironmentType {
+    Conda = 'Conda',
+}
+
 /**
  * Details about a Python runtime.
  *
  * @prop path - the location of the executable file
  * @prop version - the runtime version
  * @prop sysVersion - the raw value of `sys.version`
- * @prop architecture - of the host CPU (e.g. `x86`)
  * @prop sysPrefix - the environment's install root (`sys.prefix`)
- * @prop pipEnvWorkspaceFolder - the pipenv root, if applicable
+ * @prop envType - the kind of Python environment
  */
 export type InterpreterInformation = {
     path: string;
     version?: PythonVersion;
     sysVersion?: string;
     sysPrefix: string;
+    envType?: EnvironmentType;
```

Comment:
Small change, but I think worth making, if just to stay more in sync with how core handles this (since we cross over some with the interpreter info and API). envType is mixed in by the export type PythonEnvironment type below, not here in the Interpreter info. Keeps the distinction clear between interpreter info and environment info.